doc/mgmt/Management Framework.md

```diff
@@ -651,63 +845,111 @@ REST server will accept concurrent requests. Translib provides appropriate locki
+It supports YANG library with multi data stores and data store schemas. However these features
+are not available in SONiC. Hence REST server does not implement RFC8525 YANG library APIs (for now).
 
-* No authentication
-* TLS Certificate authentication
-* Username/password authentication
+###### 3.2.2.4.8.4 RESTCONF capabilities
+
+REST server supports "GET /restconf/data/ietf-restconf-monitoring:restconf-state/capabilities" API to
+advertise its capabilities as described in [RFC8040, section 9.1](https://tools.ietf.org/html/rfc8040#section-9.1).
+REsponse includes below mentioned capability information.
+
+    urn:ietf:params:restconf:capability:defaults:1.0?basic-mode=report-all
+
+###### 3.2.2.4.9 RESTCONF Query Parameters
 
-Only one mode can be active at a time. Administrator can choose the authentication mode through ConfigDB REST_SERVER table entry. See [DB Schema](#3_2_2_4_14-db-schema) section.
+RESTCONF Query Parameters will be supported in future release. All query parameters will be ignored by REST server in this release.
+
+###### 3.2.2.4.10 RESTCONF Operations
 
-###### 3.2.2.4.12.1 No Authentication
+REST server supports invoking YANG RPCs through "POST /restconf/operations/{rpc_name}" API
+as described in [RFC8040, section 4.4.2](https://tools.ietf.org/html/rfc8040#section-4.4.2).
+Operations modeled through YANG v1.1 action statement are not supported in this release.
 
-This is the default mode. REST server will not authenticate the client; all requests will be processed. It should not be used in production.
+YGOT binding objects are not available for YANG RPC input and output data model.
+Hence payload validation is not performed by REST server or Translib for these APIs.
+App modules will receive raw JSON data.
 
-###### 3.2.2.4.12.2 Certificate Authentication
+###### 3.2.2.4.11 RESTCONF Notifications
 
-In this mode TLS public certificate of the client will be used to authenticate the client. Administrator will have to pre-provision the CA certificate in ConfigDB DEVICE_METADATA|x509 entry. REST server will accept a request only if the client TLS certificate is signed by that CA.
+RESTCONF Notification are not supported by framework. Clients can use gNMI for monitoring and notifications.
 
-###### 3.2.2.4.12.3 User Authentication
+###### 3.2.2.4.12 Authentication
 
-In this mode REST server expects the client to provide user credentials in every request. server will support HTTP Basic Authentication method to accept user credentials.
+REST server supports following authentication modes.
 
-REST server will integrate with Linux PAM to authenticate and authorize the user. PAM may internally use native user database or TACACS+ server based on system configuration. REST write requests will be allowed only if the user belong to admin group. Only read operations will be allowed for other users.
+* HTTP Basic authentication (username/password authentication)
```

Comment:
How does it integrate with HamD & tacacs/radius ?

Reply:
REST server uses ssh to host with the username and password obtained from HTTP header. This implementation is already present in community code.